CHANGELOG.md

```diff
@@ -3,15 +3,67 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.68.0
 
-- allow OpenTelemetry resource attributes to be configured under distributed_tracing config
+This release contains a mix of features and bugfixes.
+
+### Breaking Changes
+
+#### `entrypoint` annotation implies `document` scope
+
+The [entrypoint annotation's](https://www.openpolicyagent.org/docs/latest/policy-language/#entrypoint) scope requirement 
```

Comment:
This is good! I'd perhaps add some about the motivation for this, like:

> The [entrypoint annotation's](https://www.openpolicyagent.org/docs/latest/policy-language/#entrypoint) scope requirement for rules has changed from `rule` to `document` ([https://github.com/open-policy-agent/opa/issues/6798](#6798)). Furthermore, if no `scope` annotation is declared for a METADATA block preceding a rule, the presence of an `entrypoint` annotation with a `true` value will assign the block a `document` scope, where the `rule` scope is otherwise the default.

> In practice, a rule entrypoint **always** point to the entire document and not a particular rule definition. The previous behavior was a bug, and one we've now addressed.